### PR TITLE
Trim README.md and link to our website more prominently.

### PR DESCRIPTION
**Pull request by ScottTodd**
* Remove "Getting Started" section entirely, since the new website covers that from a few different angles
* Remove "Architecture and Goals" section entirely. The new homepage covers these at a high level. The "Demonstrate MLIR's ability" and "Embrace standard-based ML via Vulkan" paragraphs are nice, but they need a better home :)
* Remove "Roadmap and Milestones" section entirely. It has been moved to https://google.github.io/iree/#roadmap (which then links back to GitHub... open to better ideas there)
* Reorder "Presentations and Talks"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-models to a unified IR optimized for real-time mobile/edge inference against
-heterogeneous hardware accelerators. IREE also provides flexible deployment
-solutions for the compiled ML models.
+pronounced as "eerie") is an [MLIR](https://mlir.llvm.org/)-based end-to-end
+compiler that lowers Machine Learning (ML) models to a unified IR optimized for
+real-time inference on mobile/edge devices against heterogeneous hardware
+accelerators. IREE also provides flexible deployment solutions for its compiled
+ML models.
+
+See [our website](https://google.github.io/iree/) for project details, user
```
Comment (stellaraccident) [inline README.md:10]: I wish this was more prominent, but I can't think of a better way to make that so right now...

**Review by stellaraccident — APPROVED** [review]

**Review by stellaraccident — COMMENTED** [review]
I would move Presentations and Talks to the end.

**Comment by ScottTodd** [comment]
> I would move Presentations and Talks to the end.

Under build status? Or just after communication channels? I moved it to put some space between "With that said, we welcome any kind of feedback on any communication channels!" and the section that linked to:

---

image

File: README.md (hunk 1 above)
Comment (ScottTodd) [reply README.md:10]: The website address is also included off to the side: 

image

**Comment by ScottTodd** [comment]
> > I would move Presentations and Talks to the end.
> 
> Under build status? Or just after communication channels? I moved it to put some space between "With that said, we welcome any kind of feedback on any communication channels!" and the section that linked to:
> 
> image

Ok, just moved it to the end, between build status and the license.

**Pull request merged**